analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TestClassShouldHaveTestMethod.MsTest.cs

```diff
@@ -113,6 +113,74 @@ public static void AfterTests()
         {
         }
     }
+
+    // Reproducers for https://github.com/SonarSource/sonar-dotnet/issues/5394
```

Comment:
These cases doesn't reproduce what user reported, because he didn't use `static` while all of these uses `static`.